sway/commands/output/background.c

```diff
@@ -63,6 +63,12 @@ struct cmd_results *output_cmd_background(int argc, char **argv) {
 
 		wordexp_t p;
```

Comment:
I was testing this and you need to initialise this to `(0)`, because if an error occurs, the values do not get initialised and so `wordfree` gets called on junk data, e.g. a segfault occurs if you run `swaymsg 'output * bg \"'`. (Technically outside the scope of this PR but since it's such a simple change I thought it could be thrown in there since it's related anyway)

Reply:
Done